packages/terra-worklist-data-grid/src/subcomponents/ColumnHeaderCell.module.scss

```diff
@@ -43,11 +51,8 @@
   }
 
   .pinned {
-    border: var(--terra-worklist-data-grid-pinned-columns-header-border, 1px solid #000);
-    border-width: 1px 0;
-    box-shadow: inset var(--terra-worklist-data-grid-pinned-columns-header-box-shadow, 1px 0 #000);
     position: sticky;
-    z-index: 2;
+    z-index: 2 !important;
```

Comment:
Why do we need the `!important` rule here.

Reply:
I wanted to guarantee that this rule wins over the other z-index rules.